ddsc/cmdparser.py

```diff
@@ -354,6 +355,9 @@ def register_upload_command(self, upload_func):
         add_project_name_or_id_arg(upload_parser, help_text_suffix="upload files/folders to.")
         _add_folders_positional_arg(upload_parser)
         _add_follow_symlinks_arg(upload_parser)
+        upload_parser.add_argument("--existing-files-full-compare",
```

Comment:
I don't think "full compare" is accurate, I'd suggest `--always-compare-checksums`